pkg/inventory/containers.go

```diff
@@ -18,6 +18,10 @@ func getRegistryOverrideNormalisedImageTag(imageTag, missingRegistryOverride str
 	if missingRegistryOverride != "" {
 		parts := strings.Split(imageTag, "/")
 		if len(parts) <= 2 {
+			// Check if the first part is a registry by seeing if it is a domain
+			if len(parts) > 1 && strings.Contains(parts[0], ".") {
```

Comment:
Could probably also check for `:` which would indicate a port was being provided.

Reply:
ah good idea I'll add that quickly